### PR TITLE
Add sleep in order to let data mapper fully load

### PR DESCRIPTION
**Pull request by jsafarik**
//test: `@integrations-odata-create`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
OData Connector | Create entity in OData service 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java [hunk 1]
```diff
@@ -137,6 +137,12 @@ public void createMapping(String source, String target) {
      * This method opens all collection mappings we can find on a data mapper
      */
     public void openDataMapperCollectionElement() {
+        try {
+            log.info("Waiting for Data Mapper to fully load before opening all collection mappings");
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
         $$(Element.MAPPER_COLLECTION_ICON).forEach(e -> {
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java]: If you are waiting for something, do not use static sleep but dynamic TestUtils.waitFor...

File: ui-tests/src/test/resources/features/integrations/odata.feature [hunk 2]
```diff
@@ -206,20 +206,20 @@ Feature: OData Connector
     When sleep for "5000" ms
     Then validate that OData service contains entity with "<key>":"<value>" property:value pair in "<resource_collection>" collection
 
-  @gh-5386
+    @gh-5386
```
[the remark this answers is not in the document]
Comment (jsafarik) [reply ui-tests/src/test/resources/features/integrations/odata.feature]: As far as I know, it does not. It seems this issue was resolved earlier then our migration to Jira.

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by jsafarik** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - DB to DB | Read and update operations 
OData Connector | Create entity in OData service 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**